### ISSUE TITLE
Which structure for the samples and libraries collection?

### ISSUE THREAD

**Issue by AdrienTorris** [AdrienTorris/awesome-blazor#133]
For now the samples and the libraries collections are two simple lists, I'm sure that we could do better to help the community to find the needed item.

What would be the perfect structure to sort the samples and libraries? Any idea?

**Comment by stsrki** [comment]
I thinks you could split the Libraries & Extensions and create more specific categories. Also I think every category could have some short description on what it could contain and what it is used for.

eg.

## Components

This will contain reusable components like buttons, inputs, etc

- Some Bootstrap library
- Some Material library
- Toast

## Tools (or Utilities)

Libraries for state management, cookies, local storage etc, basically some specific tools

- Storage
- Redux something
- Cookie something

## Libraries

This could be generic category for libraries that are not having some specific purpose.

- lib 1
- lib 2
- etc

Samples could be also divided into Games, Business apps, Tools, Todo apps. Also the ones that have **live demo** should maybe be on top of the list.

**Comment by AdrienTorris** [comment]
Thank's for your feedback @stsrki.

**Comment by AdrienTorris** [comment]
I started to reorganized the items to provide more clarity. It's a work in progress. Feel free to share your feelings or ideas!

**Comment by stsrki** [comment]
It looks good. Maybe there could be some more specific subcategories in the future but for now this is perfect.

**Comment by AdrienTorris** [comment]
Yes I agree for more specific subcategories, it just a start. Thank you so much for your feedback, it was extremely valuable 👍

**Comment by AdrienTorris** [comment]
I'd like find a name for a subcategory in the sample to group the 'technological experiments' like Blazor and Electron, Blazor and Xamarin and Blazor and Sitecore.
Any idea?

**Comment by stsrki** [comment]
_Experiments_ could be OK but the problem is that almost all of the projects are basically still experiments. Maybe better is _Proof of Concept_ or _Proof of Prototype_, something along those lines. Or to make it shorter: _Concepts_ or _Prototypes_. Whatever you choose there will still be a short description so there is no misunderstanding of what the subcategory actually represents.
